settings.gradle

```diff
@@ -7,6 +7,7 @@ include 'tritium-jmh'
 include 'tritium-lib'
 include 'tritium-proxy'
 include 'tritium-metrics'
+include 'tritium-metrics-jvm'
```

Comment:
This module depends on dropwizard-metrics `metrics-jvm`, however that is relatively small and provides no additional dependencies.
Thoughts on the separate module vs adding the dependency on metrics-jvm to `tritium-metrics`?

Reply:
I don't mind either way, don't see a problem with separating things more like this, since in practice we'll only take a direct dependency on these from a couple of internal projects.

Reply:
Agree with @dansanduleac. The separation seems fine